projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,643 @@
+  ],
+  "inputs": [
+    {
+      "tx_id": "00000000caaa37ab729805b91af2de8174e3ef24410f4effc4ffda3b610eae65",
+      "index": 1,
+      "data": "RjBEAiAYR8jc+zqY596QyMp+K3Eag3kQB5aXdfYja19Fa17u0wIgCdhBQpjlBiAawP/9WRAqAzW85CJlBpzq+YVhUALg8IUhAueFQuEkAo+s2m7nj/hnh0nyphcUuxa2LoRBjOsEOHRQ"
+    },
+    {
+      "tx_id": "00000000caaa37ab729805b91af2de8174e3ef24410f4effc4ffda3b610eae65",
+      "index": 2,
+      "data": "RzBFAiEAofVXnCKNCEu4GRk7j+wHpQM6qmezRcfxHCe/PcUdbegCIE2nip27ZQtkpkEgNEhycqHM4CkLYMLVUgskphYsd/M9IQLHG6YJxXifQ6eMxPHbINFEJAUvrzKWe9V7AXXW4iywjg=="
+    }
+  ],
+  "outputs": [
+    {
+      "value": 100,
+      "token_data": 0,
+      "script": "dqkUqdK8VisGSJuNItIBRYFfSHfHjPeIrA=="
+    },
+    {
+      "value": 200,
+      "token_data": 0,
+      "script": "dqkUISAnpOn9Vo269QBvOfBeWJTLx82IrA=="
+    }
+  ],
+  "tokens": []
+}
+```
+
+**`op_status`**
+
+This RPC method will yield different `op_status` when the `htr_getOperationStatus`
+is called depending on the current state. It might be one of:
+
+1. `WAITING_USER_INPUT` - The wallet is waiting for user confirmation
+2. `SELECTING_UTXOS` - The wallet is fetching for utxos that match the requested
+   transaction outputs
+3. `CREATING_TX_PROPOSAL` - **Specific to the wallet-service facade** The wallet
+   is creating the TxProposal on the wallet-service
+4. `MINING_TX` - The wallet is mining the transaction
+5. `PUSHING_TX` - The wallet is pushing the transaction to its connected
+   fullnode
+
+
+### 2.1.2 `htr_createToken`
+
+The `htr_createToken` method creates a new TokenCreationTransaction which will
+create a new token in the blockchain
+
+**Parameters**
+
+1. `name` - The name of the new token
+2. `symbol` - Symbol of the token
+3. `amount` - The amount of tokens to mint as an integer
+4. `address` - (Optional) Destination address of the newly minted tokens
+5. `change_address` - (Optional) Address to send the change amount
+6. `create_mint` - (Defaults to true) If the wallet should create a mint authority for the newly created token
+7. `mint_authority_address` - (Optional) Address to send the mint authority output
+8. `allow_external_mint_authority_address` - (Defaults to false) Flag indicating if the mint authority address is allowed to be from another wallet.
+9. `create_melt` - (Defaults to true) If the wallet should create a melt authority for the newly created token
+10. `melt_authority_address` - (Optional) Address to send the melt authority output
+11. `allow_external_melt_authority_address` - (Defaults to false) Flag indicating if the melt authority address is allowed to be from another wallet.
+12. `push_tx` - (Defaults to `true`) Whether the client should push the transaction to the connected fullnode
+13. `network` - The network this dApp is expecting the client to be. The request
+   will fail if the client's network is different from this params value
+
+**Example Request:**
+
+```json
+{
+  "id": 3,
+  "jsonrpc": "2.0",
+  "method": "htr_signWithAddress",
```

Comment:
Wrong method

Reply:
Fixed in 14a7fa31876e1ca55bd63a3eb57fd26a98d43cd5